packages/server/test/unit/browsers/chrome_spec.js

```diff
@@ -39,7 +40,7 @@ describe('lib/browsers/chrome', () => {
       this.onCriEvent = (event, data, options) => {
         this.criClient.on.withArgs(event).yieldsAsync(data)
 
-        return chrome.open('chrome', 'http://', options, this.automation)
+        return chrome.open({ isHeadless: true, isHeaded: false }, 'http://', options, this.automation)
```

Comment:
Why are both needed? Is one of these fine?

Reply:
Good call. I updated this.